### PR TITLE
RF Feed Bug

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/40514

## Description
Same indicator with small differences were fetched from RF Feed when connectAPI and fusion were enabled. Resulting in a timeline change for such indicator every fetch, causing index DB to explode

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by tomneeman151293** [comment]
test failure
https://app.circleci.com/pipelines/github/demisto/content/113042/workflows/2c65e587-104d-43c4-9384-cc6c8eeb92f7/jobs/422378

**Review by dantavori — APPROVED** [review]
niice!

**Comment by tomneeman151293** [comment]
@dantavori Following a small session with RF team, I will add to this PR some documentations as well regarding recommended configurations

File: Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture_description.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+## Important
```
Comment (bakatzir) [inline Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture_description.md:1]:
```suggestion
## Notes
```

File: Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/README.md [hunk 2]
```diff
@@ -26,7 +26,8 @@ This integration was integrated and tested with Recorded Future Feed
     | Malicious Threshold | The minimum score from the feed in order to to determine whether the indicator is malicious. Default is "65". For more information about Recorded Future scoring go to integration details. | False |
 
 4. Click **Test** to validate the URLs, token, and connection.
-
+## Important
+It is highly recommended to not create multiple instances of the same indicator type, even when fetching both from fusion and connectApi. Creating multiple instances with same indicator type will lead to duplicate indicators being fetched which can cause performance issues for the server.
```
Comment (bakatzir) [inline Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/README.md:30]: please move under notes

File: Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py [hunk 3]
```diff
@@ -340,6 +340,9 @@ def fetch_indicators_command(client, indicator_type, limit: Optional[int] = None
             for item in feed_dicts:
                 raw_json = dict(item)
                 raw_json['value'] = value = item.get('Name')
+                if value in indicators_value_set:
+                    continue
```
Comment (bakatzir) [inline Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py:344]: if you will add an existing to a set, nothing will change. what am I missing here?

Comment (bakatzir) [reply Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py:344]: but it will save run time

**Comment by tomneeman151293** [comment]
@dantavori Added more docs, see last commit

File: Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/README.md [hunk 4]
```diff
@@ -26,13 +26,22 @@ This integration was integrated and tested with Recorded Future Feed
...
+1. It is highly recommended to not create multiple instances of the same indicator type, even when fetching both from fusion and connectApi. Creating multiple instances with same indicator type will lead to duplicate indicators being fetched which can cause performance issues for the server.
+2. Recommended interval for fetching indicators according to Recorded Future documentation:
+
+    | **Indicator Type** | **Recommended Fetch Interval**
+    | --- | --- |
+    | IP | 1 Hour. |
+    | Domain | 2 Hours. |
+    | Hash | 1 Day. |
+    | URL | 2 Hours. |
+3. It is recommended to use either `connectApi` or `fusion` as a service, as most of the data between both is duplicated.
```
Comment (dantavori) [inline Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/README.md:39]: Did you mean here not to use both services on the same indicator type? please emphasize.

**Comment by synth-humbert** [comment]
This is this version of Recorded Future, correct?
1.0.14 - 7567740

**Pull request merged**